CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## 2.2.1 - 2023-08-10
+### Changed
```

Comment:
```suggestion
### Added
- Support for reporting the stack trace in case of error events, if provided by the player

### Changed
```

I think we're missing a feature here 🙂

Reply:
Thanks for reminding that.